docs/rst/knora-api-server/design-documentation/authentication.rst

```diff
@@ -26,146 +26,139 @@ Authentication in the Knora API Server
 Scope
 ------
 
-Authentication is the process of making sure that if someone is
-accessing something then this someone is actually also the someone
-he pretends to be. The process of making sure that someone is
-authorized, i.e. has the permission to access something, is handled as
-described in the section on authorization in the Knora base ontology
-document. TODO: add a link to this.
+Authentication is the process of making sure that if someone is accessing something then this someone is actually also
+the someone he pretends to be. The process of making sure that someone is authorized, i.e. has the permission to access
+something, is handled as described in the section on authorization in the Knora base ontology document. TODO: add a link to this.
 
 Implementation
----------------
+--------------
 
-The authentication in Knora is based on Basic Auth
-`HTTP basic authentication`_, URL parameters, and cookies. This means that
-on every request (to any of the routes), an authentication header, URL
-parameters or cookie need to be sent.
+The authentication in Knora is based on Basic Auth `HTTP basic authentication`_, URL parameters, `JSON Web Token`_ and
+cookies. This means that on every request (to any of the routes), credentials need to be sent either via authorization
+header, URL parameters or cookie header.
```

Comment:
Does this mean that a JSON web token can be sent in an authorization header, in a URL parameter or in a cookie header?

Reply:
Yes.